kas/machine/welotec/arrakis-pico.yaml

```diff
@@ -0,0 +1,19 @@
+# kas config file
+
+header:
+  version: 14
+
+repos:
+  ext/meta-yocto-bsp:
+    url: "https://git.yoctoproject.org/poky"
```

Comment:
to me it's clear that we would like to avoid maintaining the x86 stuff ourselves, but isn't it rather dirty to use poky distro while actually implementing an own distro?
a sign that it really might be this way is the required patch to get rid of poky's OE initialization script below.
not that I would have a better solution right now, just thinking ...

Reply:
ok for me